### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -10064,6 +10071,7 @@ oterwice->otherwise
 oterwise->otherwise
 oterwize->otherwise
 othe->other
+other wise->otherwise
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: whitespace

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -10114,6 +10114,7 @@ ourselve->ourselves
 ourselvs->ourselves
 ouside->outside
 oustanding->outstanding
+out-of-bound->out-of-band
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: https://en.wikipedia.org/wiki/Out_of_bounds

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: @Gelma the commit text needs to be amended as well 
image

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: image

**Pull request closed without merging**